### PR TITLE
Feature: One-Dark Theme

### PR DESCRIPTION
**Pull request by pbnj**
Add pixel-perfect theme of the Atom default `One Dark` UI & Syntax.

### Screenshot
screen shot 2016-12-30 at 2 21 17 pm

### CODE REVIEW

**Comment by Fred-Barclay** [comment]
@pmbenjamin I'd love to have this in my fork of platformio-terminal: https://github.com/Fred-Barclay/Termination
Would you mind opening a pull request there as well? I'll be sure to merge you in.
Thanks!

**Comment by pbnj** [comment]
@Fred-Barclay may I ask what's the difference between this plugin and yours?

**Comment by Fred-Barclay** [comment]
@pmbenjamin no worries. My terminal is designed to look like the old terminal-plus package (which no longer seems to be maintained). Basically, when the platformio devs forked terminal-plus to platformio-terminal, they changed the appearance to fit better with PlatformIO. I more or less undid those changes (for users who preferred the appearance of terminal-plus) while keeping nearly all of the platformio-terminal codebase.

TL;DR: My package has the appearance of an older package but uses code from a newer package. :smile:

**Comment by pbnj** [comment]
@Fred-Barclay , thank you for the info. It would certainly be nice if detailed differences are listed in your project as I'm curious about these differences.

Having said that, the only visible difference I see is the location of the terminal session tabs or the terminal bar at the bottom of the main editor window, as opposed to the bottom of the app window.
I seem to like where it is located in Platformio's plugin, but I will contribute the theme, nonetheless.

**Comment by Fred-Barclay** [comment]
@pmbenjamin Certainly the biggest change is the location of the terminal bar. My goal was to create a more traditional interface (I personally can't get used to the PlatformIO style) though I'm definitely planning on adding features (and contributing them back here) as time allows. :smile: It's all just a matter of tastes, really.

Listing the differences sounds like a great idea and something I really need to do!

**Comment by ivankravets** [comment]
Thanks! Happy New Year! 🎄
